OpTestConfiguration.py

```diff
@@ -315,6 +315,8 @@ def get_parser():
         'git repo', 'Git repository details for upstream kernel install/boot')
     gitgroup.add_argument(
         "--git-repo", help="Kernel git repository", default=None)
+    gitgroup.add_argument(
+        "--git-repo-path", help="Linux source tar file path", default=None)
```

Comment:
What are we supposed to provide as an argument? The parameter name suggests it should be a git repo and the help text says it should be a tarball.